clang/include/clang/Sema/Sema.h

```diff
@@ -3548,6 +3561,56 @@ class Sema final : public SemaBase {
     BuiltinFunction
   };
 
+  /// A helper function to provide Attribute Location for the Attr types
```

Comment:
Is this comment right?  ParsedAttr will always call the non-template version below, right?

Reply:
Yes, I moved them together, so http://eel.is/c++draft/over.match.best#general-2.4 should apply as usual.

Reply:
Right, my concern is the comment claims it is for both, but I see you copied it over, so I probably wrote the misleading comment :)

Reply:
We decided that there's nothing to change here after an offline discussion.